act/headings.md

```diff
@@ -0,0 +1,137 @@
+# Headings
+
+## Rule Identifier
+
+gel-foundations-headings
+
+## Rule Description
+
+Headings provide a semantic and visual hierarchical structure to a document.
+
+This rule tests that pages have a heading structure meets the requirement of BBC GEL Headings.
```

Comment:
"have a heading structure **that** meets..."

Reply:
```suggestion
This rule tests that pages have a heading structure that meets the requirements of BBC GEL Headings.
```